### PR TITLE
bugfix: Object editor throw error in ie10 and multiple editor create the links twice

### PR DESCRIPTION
**Pull request by hybtalented**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks backward-compatibility?    | ❌
| Tests pass?   | ✔️/❌
| Fixed issues  | #599 
| Updated README/docs?   | ❌
| Added CHANGELOG entry?   | ❌
1.  The multiple editor should not create the links, the links will be create by the child editor;
2. The second param of `insertBefore` should be an exist item (for ie10), or it will throw an exception.

### CODE REVIEW

File: src/editors/object.js [hunk 1]
```diff
@@ -758,7 +758,7 @@ export var ObjectEditor = AbstractEditor.extend({
         self.toggleEditJSON()
       })
       this.controls.appendChild(this.editjson_control)
-      this.controls.insertBefore(this.editjson_holder, this.controls.childNodes[1])
+      this.controls.insertBefore(this.editjson_holder, this.controls.childNodes[0])
```
Comment (germanbisurgi) [inline src/editors/object.js]: What does this change? and why?

I cloned the original repo and tested with ie10 but did not experienced any error

Comment (hybtalented) [reply src/editors/object.js]: When **object editor** create `editjson_holder` to `controls`, the `control` only have 1 child node, but this line try to insert  `editjson_holder` before second child node (which actually not exist).

I test it at ie version of  `10.0.9200.17556`.

Comment (schmunk42) [reply src/editors/object.js]: How is the situation for other browsers?

Comment (hybtalented) [reply src/editors/object.js]: > How is the situation for other browsers?

The exception only ocurr in native ie10(or lower), I test in ie10  mode of ie11 and chrome, old version works all right.

**Comment by schmunk42** [comment]
Thank you!

**Pull request merged**